node_modules/oae-zencoder/lib/rest.js

```diff
@@ -0,0 +1,50 @@
+/*!
+ * Copyright 2014 Apereo Foundation (AF) Licensed under the
+ * Educational Community License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License. You may
+ * obtain a copy of the License at
+ *
+ *     http://opensource.org/licenses/ECL-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an "AS IS"
+ * BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+var OAE = require('oae-util/lib/oae');
+var OaeServer = require('oae-util/lib/server');
+var log = require('oae-logger').logger('zencoder-rest');
+
+var ZencoderAPI = require('./api');
+
+OaeServer.addSafePathPrefix('/api/zencoder/encodingNotification');
+
+/**
+ * @REST postZencoderNotification
+ *
+ * Send notification when encoding job is finished
+ *
+ * @Api         private
+ * @Server      tenant
+ * @Method      POST
+ * @Path        /zencoder/encodingNotification
+ * @BodyParam   {notification}    body         Upload metadata from Zencoder
+ * @Return      {void}
+ * @HttpResponse                  200          Encoding completed successfully
+ */
+OAE.tenantRouter.on('post', '/api/zencoder/encodingNotification', function(req, res) {
+    // Tell Zencoder we received the notification or they'll try to resend it
+    res.send(202, {message: 'Notification received'});
+
+    ZencoderAPI.encodingCompleted(req.body, function(err) {
+        if (err) {
+            log().error('Could not save video previews', err);
```

Comment:
Is the second parameter contributing anything to the log message? That said, I don't think at this point we have enough information to give a meaningful log message. We only have an opaque request body that we don't necessarily understand inside the rest api. We should probably let ZencoderAPI do the logging since it knows how to extract the content item id, tenant alias and such for a useful log message.